engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -127,11 +127,15 @@ public void swapReadWriteBuffers() {
         buffersSwapped = !buffersSwapped;
     }
 
-    public FBO getPrimaryBufferUrn() {
+    public FBO getPrimaryBuffer() {
```

Comment:
Okay, so I created a new SwappableFBO class, that should make this a bit more generic. However, since FBOManagers only store FBOs right now (and not SwappableFBOs), I *had* to introduce a `getGBuffer()` method that returns the only SwappableFBO (for now). That being said, I addressed your concerns of the names being more clear.